README.md

```diff
@@ -33,7 +33,7 @@
   - [📝 License ](#-license-)
 
 
```

Comment:
- [ ] Great work so far 👍 but it would be great if you also change the information in the `Install`, `Usage` and `Tests` to make your readme descriptive and professional as requested by the previous reviewer 😊

Reply:
@danmainah @HAMMAS-SALEEM I have updated the file to make the readme according to requirement.